lib/test_data/config.rb

```diff
@@ -97,7 +97,7 @@ def after_rails_fixture_load(callable = nil, &blk)
     end
 
     def database_yaml
-      YAMLLoader.load_file(database_yaml_full_path)
```

Comment:
🤔 I thought my changes in that file fixed this issue - but perhaps I missed something.

`YAMLLoader` is just wrapping `YAML`, and forcing aliases in the event of it needing to.

https://github.com/testdouble/test_data/blob/main/lib/test_data/yaml_loader.rb#L3-L5

The `YAMLLoader.load_file` method doesn't take a second argument.

It's still not working on your end?

Reply:
No. Which is why I added the flag.

Reply:
Def using the gem release.

I've dropped this gem for now b/c I'm using SQLite and doing a full extraction for an alternate DB adapter is beyond what I have time for right now. Will look again later. Thanks!